Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -139,12 +140,8 @@ CppType idl_type_name_to_cpp_type(Type const& type, Interface const& interface)
     if (interface.callback_functions.contains(type.name()))
         return { .name = "JS::Handle<WebIDL::CallbackType>", .sequence_storage_type = SequenceStorageType::MarkedVector };
 
-    if (type.is_string()) {
-        if (interface.extended_attributes.contains("UseDeprecatedAKString"))
-            return { .name = "DeprecatedString", .sequence_storage_type = SequenceStorageType::Vector };
-
+    if (type.is_string())
         return { .name = "String", .sequence_storage_type = SequenceStorageType::Vector };
-    }
```

Comment:
Nit, but I think we always prefer curly braces
```suggestion
    if (type.is_string()) {
        return { .name = "String", .sequence_storage_type = SequenceStorageType::Vector };
    }
```

Reply:
I think we actually used to prefer no curlies for single line statements, but that's since evolved to be acceptable to be either or. Think no curlies is probably best to fit in with the rest of the code in this function